lib/atomic_time/core_ext/active_record/connection_adapters/column_methods.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'active_record'
+
+module ActiveRecord
+  module ConnectionAdapters
+    module ColumnMethods
+      def atomic_time(*args, **options)
+        args.each { |name| column(name, :integer, options) }
```

Comment:
This is leftover from when I tried to add `:atomic_type` as a direct database type which was thwarted by the fact that I would have to write code for each specific adapter. I thought about removing it, but I turned it into an alias for an `:integer` type which makes writing migrations a little more specific.